<ui version="4.0">
 <class>AnnoToolMainWindowClass</class>
 <widget class="QMainWindow" name="AnnoToolMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>760</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AnnoTool v2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
   <normaloff>:/res/annoToolLogo_16</normaloff>:/res/annoToolLogo_16</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="renderHints">
    <set>QPainter::Antialiasing</set>
   </property>
   <property name="cacheMode">
    <set>QGraphicsView::CacheBackground</set>
   </property>
   <property name="viewportUpdateMode">
    <enum>QGraphicsView::SmartViewportUpdate</enum>
   </property>
  </widget>
  <widget class="AnnoFileListWidget" name="annoFileListWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <widget class="AnnoListWidget" name="annoListWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <widget class="AnnoDataWidget" name="annoDataWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>860</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileNew" />
    <addaction name="actionFileOpen" />
    <addaction name="actionFileClose" />
    <addaction name="actionFileSave" />
    <addaction name="actionSaveCurrentImage" />
    <addaction name="separator" />
    <addaction name="actionFileImport" />
    <addaction name="actionFileExport" />
    <addaction name="separator" />
    <addaction name="actionFileExit" />
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionSettingFolders" />
    <addaction name="actionSetImageLoader" />
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpInfo" />
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actionNextImage" />
    <addaction name="actionPreviousImage" />
    <addaction name="separator" />
    <addaction name="actionProjectDetails" />
    <addaction name="separator" />
    <addaction name="actionProjectAddImage" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menu_Project" />
   <addaction name="menu_Settings" />
   <addaction name="menu_Help" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="tbGeneral">
   <property name="windowTitle">
    <string>General</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileNew" />
   <addaction name="actionFileOpen" />
   <addaction name="actionFileClose" />
   <addaction name="actionFileSave" />
   <addaction name="actionGroundtruthMode" />
   <addaction name="actionProjectAddImage" />
  </widget>
  <widget class="QToolBar" name="tbView">
   <property name="windowTitle">
    <string>View</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEnableAutoFit" />
   <addaction name="actionFitImage" />
  </widget>
  <widget class="QToolBar" name="tbTools">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLockParentAnno" />
   <addaction name="actionToolPointer" />
   <addaction name="actionToolSinglePoint" />
   <addaction name="actionToolRectangle" />
   <addaction name="actionToolPolygon" />
   <addaction name="actionToolEllipse" />
  </widget>
  <widget class="QToolBar" name="tbAnnotation">
   <property name="windowTitle">
    <string>Annotation Actions</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRemoveAnnotation" />
   <addaction name="actionZtoFront" />
   <addaction name="actionZtoBack" />
  </widget>
  <widget class="QToolBar" name="tbFilter">
   <property name="windowTitle">
    <string>Filters</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFilterEnable" />
   <addaction name="actionFilterEdit" />
  </widget>
  <action name="actionFileOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/folder_open</normaloff>:/res/folder_open</iconset>
   </property>
   <property name="text">
    <string>&amp;Open Project</string>
   </property>
   <property name="statusTip">
    <string>Open an existing annotation folder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/folder_close</normaloff>:/res/folder_close</iconset>
   </property>
   <property name="text">
    <string>&amp;Close Project</string>
   </property>
   <property name="statusTip">
    <string>Close current annotation folder</string>
   </property>
  </action>
  <action name="actionFileExit">
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/shutdown</normaloff>:/res/shutdown</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Exit AnnoTool</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/drive</normaloff>:/res/drive</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Project</string>
   </property>
   <property name="statusTip">
    <string>Save all annotations</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSettingFolders">
   <property name="text">
    <string>&amp;Folders</string>
   </property>
   <property name="statusTip">
    <string>Configure the general folder settings</string>
   </property>
  </action>
  <action name="actionHelpInfo">
   <property name="text">
    <string>&amp;Info</string>
   </property>
   <property name="statusTip">
    <string>Show information about AnnoTool</string>
   </property>
  </action>
  <action name="actionToolRectangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/tool_rectangle</normaloff>:/res/tool_rectangle</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
   <property name="statusTip">
    <string>Select the rectangle tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionToolPointer">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/tool_pointerS</normaloff>:/res/tool_pointerS</iconset>
   </property>
   <property name="text">
    <string>Pointer</string>
   </property>
   <property name="statusTip">
    <string>Select the pointer tool.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionToolPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/tool_polygon</normaloff>:/res/tool_polygon</iconset>
   </property>
   <property name="text">
    <string>Polygon</string>
   </property>
   <property name="statusTip">
    <string>Select the polygon tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionToolEllipse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/tool_ellipse</normaloff>:/res/tool_ellipse</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
   <property name="statusTip">
    <string>Select the ellipse tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="actionFileNew">
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/new</normaloff>:/res/new</iconset>
   </property>
   <property name="text">
    <string>&amp;New Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionProjectDetails">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Project &amp;Details</string>
   </property>
  </action>
  <action name="actionProjectAddImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/addImg</normaloff>:/res/addImg</iconset>
   </property>
   <property name="text">
    <string>Add &amp;Image</string>
   </property>
  </action>
  <action name="actionSetImageLoader">
   <property name="text">
    <string>Image &amp;Loader</string>
   </property>
  </action>
  <action name="actionFitImage">
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/fitImg</normaloff>:/res/fitImg</iconset>
   </property>
   <property name="text">
    <string>Fit Image</string>
   </property>
  </action>
  <action name="actionFileImport">
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionFileExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
  </action>
  <action name="actionRemoveAnnotation">
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/delete</normaloff>:/res/delete</iconset>
   </property>
   <property name="text">
    <string>Remove Annotation</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionZtoFront">
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/toFront</normaloff>:/res/toFront</iconset>
   </property>
   <property name="text">
    <string>Send to Front</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionToolSinglePoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/tool_point</normaloff>:/res/tool_point</iconset>
   </property>
   <property name="text">
    <string>Single Point</string>
   </property>
   <property name="statusTip">
    <string>Select the single point tool</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionSaveCurrentImage">
   <property name="text">
    <string>Save Current Image</string>
   </property>
  </action>
  <action name="actionGroundtruthMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/groundtruth</normaloff>:/res/groundtruth</iconset>
   </property>
   <property name="text">
    <string>Groundtruth Mode</string>
   </property>
  </action>
  <action name="actionLockParentAnno">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lock Parent</string>
   </property>
  </action>
  <action name="actionZtoBack">
   <property name="icon">
    <iconset resource="icons.qrc">
    <normaloff>:/res/toFront</normaloff>:/res/toFront</iconset>
   </property>
   <property name="text">
    <string>Send to Back</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionPreviousImage">
   <property name="text">
    <string>Previous Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgUp</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionNextImage">
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+PgDown</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionFilterEdit">
   <property name="text">
    <string>Edit Filter</string>
   </property>
  </action>
  <action name="actionFilterEnable">
   <property name="text">
    <string>En-/Disable Filter</string>
   </property>
  </action>
  <action name="actionEnableAutoFit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable Auto Fit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnnoFileListWidget</class>
   <extends>QDockWidget</extends>
   <header>AnnoFileListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnnoListWidget</class>
   <extends>QDockWidget</extends>
   <header>AnnoListWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnnoDataWidget</class>
   <extends>QDockWidget</extends>
   <header>AnnoDataWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections />
</ui>
